### ISSUE TITLE
MDN Web Docs no longer prompts to install as a custom search engine

### ISSUE THREAD

**Issue by jamestthompson3** [mdn/content#4376]
It was previously possible to get the prompt to add the MDN Web Docs site as a custom search engine in Firefox. The cause of this seems to be that the [Open Search](https://developer.mozilla.org/en-US/docs/Web/OpenSearch) link is no longer present in the `<head>` elements of the MDN Web Docs pages.

**Comment by sideshowbarker** [comment]
This seems like an issue for @mdn/core-dev?

**Comment by escattone** [comment]
@sideshowbarker Yes, thank you for the heads up! I'll transfer this to the yari repo.
